### PR TITLE
ci: Fix cli auto publishing

### PR DESCRIPTION
**Pull request by FabianLars**
This is https://github.com/tauri-apps/tauri/pull/10222 but for dev branch

### CODE REVIEW

File: .github/workflows/covector-version-or-publish.yml [hunk 1]
```diff
@@ -116,18 +116,14 @@ jobs:
         if: |
           steps.covector.outputs.successfulPublish == 'true' &&
           contains(steps.covector.outputs.packagesPublished, '@tauri-apps/cli')
-        uses: peter-evans/repository-dispatch@v1
-        with:
-          token: ${{ secrets.ORG_TAURI_BOT_PAT }}
-          event-type: publish-js-cli
-          client-payload: >-
-            {"releaseId": "${{ steps.covector.outputs['-tauri-apps-cli-releaseId'] }}" }
+        run: gh workflow run 31554138 -r dev -f releaseId=${{ steps.covector.outputs['-tauri-apps-cli-releaseId'] }}
```
Comment (lucasfernog) [inline .github/workflows/covector-version-or-publish.yml:119]:
```suggestion
        run: gh workflow run publish-cli-js.yml -r dev -f releaseId=${{ steps.covector.outputs['-tauri-apps-cli-releaseId'] }}
```

File: .github/workflows/covector-version-or-publish.yml [hunk 2]
```diff
@@ -116,18 +116,14 @@ jobs:
...
 
       - name: Trigger `tauri-cli` publishing workflow
         if: |
           steps.covector.outputs.successfulPublish == 'true' &&
           contains(steps.covector.outputs.packagesPublished, 'tauri-cli')
-        uses: peter-evans/repository-dispatch@v1
-        with:
-          token: ${{ secrets.ORG_TAURI_BOT_PAT }}
-          event-type: publish-clirs
+        run: gh workflow run 31554139 -r dev
```
Comment (lucasfernog) [inline .github/workflows/covector-version-or-publish.yml:127]:
```suggestion
        run: gh workflow run publish-cli-rs.yml -r dev
```

**Review by lucasfernog — COMMENTED** [review]
looks like we can use the workflow name https://cli.github.com/manual/gh_workflow_run

**Comment by FabianLars** [comment]
Yes we can but I didn't want to risk it tbh (in case we rename it)

Edit: or if we have new branches and lose oversight etc

**Review by lucasfernog — APPROVED** [review]

**Comment by lucasfernog** [comment]
that's what i thought too, let's hope github doesn't create a new id somehow

**Comment by FabianLars** [comment]
Oh man, that'd be just our luck

**Pull request merged**